41.md

```diff
@@ -0,0 +1,69 @@
+NIP-XX
+======
+
+Surveys / polls
+-----------------------------------
+
+`draft` `optional` `require:nip-20` `author:fernandolguevara`
+
+The `surv` & `surv-resp` tags enables users to have surveys/polls experiences on the network.
+
+
+#### Spec
+
+##### Message
+
+```
+tag: surv
+options:
+ - <multi|single> allow others to reply with one or multiple options
+ - <ttl> TTL (in seconds|timestamp) when surv expires, 0 TTL don't expire
+ - [<choice>]: up to 4 choices each limited to 25 chars  
```

Comment:
Because relays are dumb by design, we can't really force them to have a maximum chars which can be différents from one to another, same for the number of choices.

But we can add a recommendation to help the integration

```suggestion
 - [<choice>]: array of string
```

Reply:
we can rephrase the sentence to suggest clients implement the char limit, what do you think?